lib/safira/store/redeemable.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Safira.Store.Redeemable do
+  use Ecto.Schema
+  use Arc.Ecto.Schema
+  import Ecto.Changeset
+
+  alias Safira.Store.Buy
+  alias Safira.Accounts.Attendee
+
+  schema "redeemables" do
+    field(:img, Safira.Avatar.Type)
+    field(:name, :string)
+    field(:description, :string)
+    field(:price, :integer)
+    field(:redeem_limit, :integer)
+    field(:num_redeems, :integer, default: 0)
+    field(:max_per_user, :integer)
+
+    many_to_many(:attendees, Attendee, join_through: Buy)
```

Comment:
This syntax is not incorrect, It's just not consistent with the other schemas in Safira.

Reply:
I had it differently, then I ran the formatter and it changed it to this mode. So, I thought, "Who am I to judge?" 😅

Reply:
That being the case, I think formatter has won me this fight. 😄  Which doesn't invalidate the inconsistency.

Reply:
I agree with you! I think we should ignore the formatter in this case. Am I allowed to ignore it? :smile:

Reply:
I took a shortcut when editing the table in an existing migration, because my db is almost empty, so almost nothing changed for me. But if you want me to create a new migration with the necessary changes, just let me know. @jpcorreia99 @joaop21 @tsource98

Reply:
What migration file are you refering to?

Reply:
The update one